_content/papers/vohara_analayo.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Vohāra"
```

Comment:
Let's maybe add `(Transactions)` in parentheses afterwards (as this is a slightly more obscure term)